### ISSUE TITLE
NavBar Light Theme: State of active component lacks 3 to 1 contrast ratio

### ISSUE THREAD

**Issue by scurker** [dequelabs/cauldron#909]
Original Auditor URL: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/bd48e618-a390-11ed-900e-03f33794f7ed?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=12&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

Similar issue for dark mode: https://axeauditor.dequecloud.com/test-run/f6620faa-6d1e-11ec-a0b7-97f3ccf70959/issue/dce7a924-a38f-11ed-af79-6736182d54b0?sortField=identifier&sortDir=asc&page=0&activeTab=dt-issue&pageSize=50&viewMode=issues&row=13&issuesCount%5B0%5D=&issuesCount%5B1%5D=&filter%5Bgroup_id%5D%5B0%5D=42b8f3a2-ac6e-11ed-adb1-3bfa602e0928&filter%5Bgroup_id%5D%5B1%5D=df2b20c0-1317-11ed-acb1-c7cbc98656eb

URL: https://cauldron.dequelabs.com/components/NavBar

## Description
Light Theme: State of active component lacks 3 to 1 contrast ratio

Element Description : 
-NavItem 1
-NavItem 2
-NavItem 3
-NavItem 4
-NavItem 5

Location of the element: Below the "Demo" heading
Note: Toggle to Mobile mode

Focus indicator color code (On focus): #C11BDE
Outer border color code: #CCCCCC
Contrast ratio: 2.9:1
### RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

### HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

### REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

### BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.

![cauldron light mode navbar with one item focused](https://user-images.githubusercontent.com/1062039/219801147-d496d7ff-48d5-456d-a80b-da06c429d4f8.png)

![cauldron dark mode navbar with one item focused](https://user-images.githubusercontent.com/1062039/219801294-9f8dcb01-06e3-4e94-9d1d-8b0b49a29c4b.png)

**Comment by scurker** [comment]
This _might_ be invalid. Need to get clarification first.

**Comment by scurker** [comment]
This is not the active component, the active component has a accent bar on the left side of the nav item.
